recipe/bld.bat

```diff
@@ -5,4 +5,4 @@ echo udunits2_path = %SCRIPTS%\udunits2.dll >> %SITECFG%
 rmdir lib\iris\tests\results /s /q
 del lib\iris\tests\*.npz
 
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Any special reason to use `python` instead of `%PYTHON%`? That was the source on issue in the matplotlib feedstock. See https://github.com/conda-forge/matplotlib-feedstock/pull/18#issuecomment-217929120